R/rf_modeling.R

```diff
@@ -183,9 +183,15 @@ rf_modeling <- function( msnset, features, response, pred.cls, K=NULL, sel.feat=
         K <- nrow(dSet)
     num_rep <- ceiling(nrow(dSet)/K)
     cv_idx <- sample(rep(seq_len(K), num_rep)[seq_len(nrow(dSet))])
-    multiproc_cl <- makeCluster(max(1, detectCores() - 1))
+    if(is.null(cores)){
+        cores <- max(1, detectCores() - 1)
+    }
+    stopifnot(1 <= cores && cores <= detectCores())
+    multiproc_cl <- makeCluster(cores)
     clusterEvalQ(multiproc_cl, library("MSnID"))
     clusterEvalQ(multiproc_cl, library("Biobase"))
+    set.seed(42) # Set outer seed
+    clusterSetRNGStream(multiproc_cl, 21) # Set process seeds
```

Comment:
by conventions let's set seeds to 0. Also, let's move it into the argument list.